sns_ik_lib/CMakeLists.txt

```diff
@@ -65,11 +66,17 @@ if (CATKIN_ENABLE_TESTING)
   # unit tests
   catkin_add_gtest(rng_utilities_test test/rng_utilities_test.cpp)
   target_link_libraries(rng_utilities_test sns_ik sns_ik_test ${catkin_LIBRARIES})
+
   catkin_add_gtest(sns_ik_math_utils_test test/sns_ik_math_utils_test.cpp)
   target_link_libraries(sns_ik_math_utils_test sns_ik sns_ik_test ${catkin_LIBRARIES})
+
   catkin_add_gtest(sns_ik_pos_test test/sns_ik_pos_test.cpp)
   target_link_libraries(sns_ik_pos_test sns_ik sns_ik_test ${catkin_LIBRARIES})
+
   catkin_add_gtest(sns_ik_vel_test test/sns_ik_vel_test.cpp)
   target_link_libraries(sns_ik_vel_test sns_ik sns_ik_test ${catkin_LIBRARIES})
 
+  catkin_add_gtest(sns_vel_ik_base_test test/sns_vel_ik_base_test.cpp)
+  target_link_libraries(sns_vel_ik_base_test sns_ik sns_ik_test sns_ik_util ${catkin_LIBRARIES})
```

Comment:
`sns_ik_util` doesn't exist after #82

Reply:
ok - I'll rebase and then fix this

Reply:
Update: I did a merge instead so that I did not break the links in the code review. This pull request should be squashed onto master so that it doesn't look like 4+ commits.